### PR TITLE
Update changelog for release of 5.3.1

### PR DESCRIPTION
**Pull request by ced0ps**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,22 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
...
+### Fixes
+
+- How existing secrets for DB credentials work
+- DB permission template
+- Linting issue and the values.schema.yaml
+
+### Added
+
+- Security context to all relevant resources
+- Advance configuration section from upstream to README.md
```
Comment (webwurst) [inline CHANGELOG.md]:
```suggestion
- Advanced configuration section from upstream to README.md
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,22 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
...
+
+
```
Comment (ubergesundheit) [inline CHANGELOG.md:24]:
```suggestion
```

**Review by ubergesundheit — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,22 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Update
+
+This chart now tracks the Giantswarm Platform specific branch of upstream.
```
Comment (webwurst) [inline CHANGELOG.md]: Maybe link here to that branch?

**Review by webwurst — APPROVED** [review]
Minor comments, else fine for me :)

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,21 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
...
```
Comment (ced0ps) [inline CHANGELOG.md]:
```suggestion
This chart now tracks the [Giantswarm Platform specific branch](https://github.com/aquasecurity/aqua-helm/tree/5.3_Giant_Swarm) of upstream.
```

**Pull request merged**